plugins/search-backend-module-catalog/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@backstage/plugin-search-backend-module-catalog",
+  "description": "A module for the search backend that exports catalog modules",
+  "version": "1.1.4-next.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./alpha": "./src/alpha.ts",
+    "./package.json": "./package.json"
+  },
+  "typesVersions": {
+    "*": {
+      "alpha": [
+        "src/alpha.ts"
+      ],
+      "package.json": [
+        "package.json"
+      ]
+    }
+  },
+  "backstage": {
+    "role": "backend-plugin-module"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack",
+    "clean": "backstage-cli package clean"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "workspace:^",
+    "@backstage/backend-plugin-api": "workspace:^",
+    "@backstage/backend-tasks": "workspace:^",
+    "@backstage/catalog-client": "workspace:^",
+    "@backstage/catalog-model": "workspace:^",
+    "@backstage/config": "workspace:^",
+    "@backstage/plugin-catalog-backend": "workspace:^",
+    "@backstage/plugin-catalog-common": "workspace:^",
+    "@backstage/plugin-permission-common": "workspace:^",
+    "@backstage/plugin-search-backend-node": "workspace:^",
+    "@backstage/plugin-search-common": "workspace:^",
+    "@elastic/elasticsearch": "^7.13.0",
+    "@opensearch-project/opensearch": "^2.0.0",
+    "aws-os-connection": "^0.2.0",
+    "aws-sdk": "^2.948.0",
+    "elastic-builder": "^2.16.0",
+    "lodash": "^4.17.21",
+    "uuid": "^8.3.2",
+    "winston": "^3.2.1"
+  },
+  "devDependencies": {
+    "@backstage/backend-common": "workspace:^",
+    "@backstage/backend-test-utils": "workspace:^",
+    "@backstage/cli": "workspace:^",
+    "@elastic/elasticsearch-mock": "^1.0.0",
+    "@short.io/opensearch-mock": "^0.3.1",
+    "msw": "^1.0.0"
+  },
```

Comment:
Couple of bonus deps here right? 😅

Reply:
100% package copy paste leftovers, will take a look!